src/api/Main/BuisnessLayer/Shrooms.Domain/Services/Picture/PictureService.cs

```diff
@@ -41,6 +41,13 @@ public async Task<string> UploadFromStream(Stream stream, string mimeType, strin
             return pictureName;
         }
 
+        public async Task RemoveImage(string blobKey, int orgId)
+        {
+            var tenantPicturesContainer = _organizationsDbSet.Where(o => o.Id == orgId).Select(o => o.ShortName).First().ToLowerInvariant();
```

Comment:
For future improvements: it probably would make sense to move picture container formatting logic to separate reusable method. It's important, because currently this logic is duplicated in Upload and Remove logic. If you decide to change that logic in Upload and will change the format how image is saved - Remove will stop working.